python_modules/dagster/dagster/_generate/templates/PROJECT_NAME_PLACEHOLDER/requirements.txt.tmpl

```diff
@@ -0,0 +1,3 @@
+-e .
+dagster=={{ dagster_version }}
```

Comment:
What's the thinking behind doing this pin?

Reply:
for development, users will `pip install -e .` to use the latest version.
but when things are running in deployment, users will be using requirements.txt so if they'd *maybe* want to pin dagster, thats the place for them to pin.

but i think for scaffolding minimum, im leaning towards unpinning everything now.